utils/interactive_mode.py

```diff
@@ -0,0 +1,421 @@
+"""
```

Comment:
@akkuldn - I feel like this file will be extremely hard to maintain. A few suggestions:
a) Break your methods down better. Typically, methods should not be more than 10 lines. You should break the code down better.
b) When you intersperse code and strings, the strings should become configurable. If you are using the same strings in two or more places, you should be making it a variable. that way, when the hardcoded string needs to change, the QA changes them in only one spot.
c) Can you please add some unit tests for these options? Include them in the PR to help your reviewer. You can delete the tests once they approve your PR.

Reply:
a) Brokedown the methods making them smaller
b) Now having a single variable for most of the reused strings
c) Added some unit tests
Fixed